### DISCUSSION TITLE
Bugs & Things That Bugged Me About 3.4 To Rewind To 3.3

### DISCUSSION THREAD

**Discussion by TEOL2023** [audacity/audacity#5613] [Q&A]
### Bug description

1) It really bugged me to see forced Joint Stereo I don't want narrow panned almost Mono Stereo sound I want the true Stereophonic Experience from my conversions and this version took that away taking away freedom of choice as well.

2) I actually dislike the new Export Window even it doesn't auto select my khz to what I'm saving anymore for a start and putting crappy narrow panned almost Mono Joint Stereo as Stereo is false advertising I hate that format I want real Stereo I don't want anything that changes some parts of a Stereo track to Mono anywhere near my ears. I HATE JOINT STEREO!!!!! Make your track -6 in real Stereo not Joint Stereo with max not clipping loudness I say.

3) FMPEG crashed on an MP3 export and it wiped out both MP3 and FFMEG from the software never to be able to read then again to open videos or mp3s by then not even on clean reinstalls and even clean FFMPEG Reinstalls which wiped out Audacity upon install another glitch or either manually get them back on like you used to only a revert fixed them.

### Steps to reproduce

1. Import an MP4
2. FFMPEG Crashes on Export to MP3
3. The software stops FFMPEG detection.

### Expected behavior

Being able to choose real stereo as a export format.

Automatically detecting my imported files KHZ upon export.

Being able to manually locate FFMPEG and MP3 again.

### Actual behavior

A enforced crappy Joint Stereo format I don't want as I'd rather save everything on it's full width keeping the tracks sound integrity with a lower volume rather than a format that makes some parts Mono to work with max non clipping loudness.

44 KHZ saving as 48 KHZ all the time

FFMPEG crash once on an MP4 within my Vorbis and MP4 files and wiped out both MP3 and FFMPEG.

Took a picture of my proudly downgraded version to be able to submit my report.
![Untitled](https://github.com/audacity/audacity/assets/150247576/71ac21fc-ca10-41ff-9086-afe93a712755)

![Untitled](https://github.com/audacity/audacity/assets/150247576/7929bbaa-e4f1-4753-a957-bed14003fc62)

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

Windows

### Additional context

Please do fix everything freedom of choice format export wise

Took a picture of my proudly downgraded version to be able to submit my report.

**Comment by petersampsonaudacity** [comment]
1.  I, too, miss the ability to specify true stereo for MP3 export

3. I get no crash when I import an MP4 and export as MP3 - tested on W10 with 3.4.0 and the test build for 3.4.1

2. I love the new export window - I have no problem with Audacity not using the sample rate I specify.  Except for multiple export - there is a bug that always exports thes as 44100 regardless of the user setting.  Bit this has been fixed for the impending 3.4.1

**Comment by crsib** [comment]
> I, too, miss the ability to specify true stereo for MP3 export

In joint stereo mode, LAME selects the best option for each MPEG frame (a frame contains 1152 samples). That is - if "true" stereo yields the best quality - the frame will use it. Using this mode is a way to get the best possible quality from the encoder.

**Comment by TEOL2023** [comment]
I've had a change of heart after using it for a week after .1 was out fixing FFMPEG and having forced to use Joint this last week I hear no difference with the Stereo as such to be honest.

I still have a problem with it changing from Left and Right panning to Mid and Side panning though as it has the potential to mess up the original sound of some tunes as of course stereo sound is the Left and Right or Side only so the bits sent to mid panning all sound weird.

That is only reason why I poo pooed the forcing of the mode to be honest.
